src/main/java/eu/dissco/backend/repository/DigitalMediaObjectRepository.java

```diff
@@ -72,6 +58,13 @@ public List<String> getDigitalMediaIdsForSpecimen(String id) {
         .fetch(Record1::value1);
   }
 
+  public List<DigitalMediaObject> getDigitalMediaForSpecimenObject(String id) {
```

Comment:
What is the difference between this and `getDigitalMediaForSpecimen` is this method still needed/used